loop_lab.js

```diff
@@ -0,0 +1,299 @@
+// # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable
+// "num" and logs all the numbers, descending, between "num" and 1.
+
+let num = 10;
+
+while (num > 0) {
+  console.log(num);
+  num--;
+}
+
+for (let num = 10; num > 0; num--) {
+  console.log(num);
+}
+
+// 2. Write a `while` loop and a `for` loop that takes the variable
+// "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is
+// even or odd, and log that to the screen (e.g. "2 is even")
+
+let max = 10;
+
+while (max > 0) {
+  if (max % 2 === 0) {
+    console.log(max + ' is even');
+  } else if (max % 2 === 1) {
+    console.log(max + ' is odd');
+  } else {
+    console.log('error ' + max + ' is not an integer.');
+  } max--;
```

Comment:
I think the question was asking for ascending numbers from max, not descending from max. Minor thing really. Good implementation of while and for loops.